diagnostics/networking.sh

```diff
@@ -27,5 +27,8 @@ traceroute 1.1.1.1
 cat /etc/resolv.conf
 
 # Validate that everything is functionning correctly
-curl -m 5 -v https://microsoft.com
-
+if which curl > /dev/null; then
+    curl -m 5 -v https://microsoft.com
+else
```

Comment:
should we have an else if for wget, and an else that prints an error?

Reply:
We don't do `set -e` here so I'm ok letting bash printing `wget not found` here